backend/app/Providers/AuthServiceProvider.php

```diff
@@ -15,6 +17,7 @@ class AuthServiceProvider extends ServiceProvider
      */
     protected $policies = [
         // 'App\Models\Model' => 'App\Policies\ModelPolicy',
+        User::class => UserPolicy::class
```

Comment:
I didn't realize this was a step for applying policies.  I like it. (It had been feeling a little too "magic" for my taste)

Reply:
Your feeling wasn't wrong! https://laravel.com/docs/8.x/authorization#policy-auto-discovery